types/int_test.go

```diff
@@ -69,7 +69,7 @@ func (s *intTestSuite) TestIntPanic() {
 	s.Require().Panics(func() { i2.Neg().Mul(i2.Neg()) })
 	s.Require().Panics(func() { i3.Neg().Mul(i3.Neg()) })
 
-	// Underflow check
+	// // Underflow check
```

Comment:
By the way, this is not an underflow

> Storing values that are too low in an integer variable (e.g., attempting to store −1 in an unsigned integer) is properly referred to as integer overflow, or more broadly, integer wraparound. The term underflow normally refers to floating point numbers only, which is a separate issue.

https://en.wikipedia.org/wiki/Arithmetic_underflow